### DISCUSSION TITLE
Unable to test project in Sandbox

### DISCUSSION THREAD

**Discussion by ff1e42** [spatialsys/spatial-unity-sdk#15] [Help] [answered]
Hi everyone, is anyone else experiencing problems when trying to test a unity project in Spatial.io Sandbox?
I just downloaded the spatial-unity-starter-template-main folder, extracted it, opened it as a new project in Unity using the 2021.3.21f1 editor version, then imported a simple fbx from blender, set the entrance point and tried to test it, but when I do so I get these error messages:
![sandbox error 2](https://github.com/spatialsys/spatial-unity-sdk/assets/162799994/0177b94d-f122-4f9f-8ff3-24509271a237)
![sandbox error 1](https://github.com/spatialsys/spatial-unity-sdk/assets/162799994/d3ccc724-3602-49d7-b533-c1e7b6769b48)

**Comment by AlanSpatial** [comment, marked as answer]
Open the spatial portal:
![image](https://github.com/spatialsys/spatial-unity-sdk/assets/100000804/758930c8-7060-4674-a04f-c95d0e9e5ceb)

And make sure your `active package` is set to the scene you are testing. You may need to create a new package.
![image](https://github.com/spatialsys/spatial-unity-sdk/assets/100000804/4a65c225-97ca-42ab-b790-ecf8b2ad1e76)


https://docs.spatial.io/project-configuration
